cves/CVE-2013-0305.yml

```diff
@@ -217,8 +254,10 @@ lessons:
     applies:
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      The vulnerability directly deals with the ability of users to access the resources necessary for their role. In this
+      case, emphasizing that there are specific roles that require no one has access to these resources. 
```

Comment:
I did not see this in the fixes you mentioned... Was the history view not supposed to be viewed by anyone or did it just require additional authentication?

Reply:
The history view was supposed to only be accessed by those with admin privileges. The principle of least privileges states that modules (users) can only access data that is necessary for their purpose, and in this case the data some users could get from the history view was beyond the purpose of their role.